### ISSUE TITLE
Variant name validation failed

### ISSUE THREAD

**Issue by HolQue** [test-fullautomation/robotframework-testsuitesmanagement#125]
In previous versions of the testsuites management the following use case was successful. Now this use case fails.

I use configuration files including a variant with special characters inside the name:

```
"SälfTest.ß.€.考.𠼭.𠼭": {
             "name": "exercise-pg_config_SälfTest.ß.€.考.𠼭.𠼭.json",
             "path": ".../config/"
            }
```

I refer to this variant in command line with:

`--variable variant:"SälfTest.ß.€.考.𠼭.𠼭"`

Outcome:

```
[ ERROR ] Testsuite management - Loading configuration level 2 failed!
          The variant name 'SälfTest.ß.€.考.𠼭.𠼭' is invalid.
          Please find the suitable variant in this list:
            - 'default'
            - 'variant1'
            - 'variant2'
            - 'SälfTest.ß.€.考.𠼭.𠼭'
```

A cross check with

`--variable config_file:"./config/exercise-pg_config_SälfTest.ß.€.考.𠼭.𠼭.json"`

is successful. Seems that the variant name validation has an issue.

**Comment by namsonx** [comment]
Hello Holger,

I created the pull-request #130 to fix this issue.

Thank you,
Son

**Comment by HolQue** [comment]
Works fine now.
(retest with latest available feature branches / 28.02.2023)

**Comment by test-fullautomation** [comment]
solved and already merged with #130
